Gems/Atom/RPI/Code/Source/RPI.Public/Image/ImageSystem.cpp

```diff
@@ -347,6 +389,14 @@ namespace AZ
                 Data::AssetId m_assetId;
             };
 
+            // Sync values from desc back to the cvars
+            // Note: we need this code because one of the instance of the cvar might be initialized early than setting registry
+            // due to this issue: https://github.com/o3de/o3de/issues/5537
+            AZ::CVarFixedString commandString = AZ::CVarFixedString::format("r_streamingImagePoolBudgetMb %d", desc.m_systemStreamingImagePoolSize);
```

Comment:
The percent specifier here needs to be "%" PRiu64 since `m_systemStreamingImagePoolSize` is a [uint64_t](https://en.cppreference.com/w/c/types/integer#Format_macro_constants)
```suggestion
            AZ::CVarFixedString commandString = AZ::CVarFixedString::format("r_streamingImagePoolBudgetMb %" PRIu64, desc.m_systemStreamingImagePoolSize);
```